sigar_linux.go

```diff
@@ -137,6 +137,103 @@ func (self *CpuList) Get() error {
 
 	return err
 }
+func (self *NetProtoV6Stats) Get() error {
+	return readFile(Procd+"/net/snmp6", func(line string) bool {
+		fields := strings.Fields(line)
+		if len(fields) != 2 {
+			return true
+		}
+		switch fields[0] {
+		case "Ip6InReceives":
+			self.IP.InReceives, _ = strtoull(fields[1])
+		case "Ip6InAddrErrors":
+			self.IP.InAddrErrors, _ = strtoull(fields[1])
+		case "Ip6OutForwDatagrams":
+			self.IP.ForwDatagrams, _ = strtoull(fields[1])
+		case "Ip6InDelivers":
+			self.IP.InDelivers, _ = strtoull(fields[1])
+		case "Ip6OutRequests":
+			self.IP.OutRequests, _ = strtoull(fields[1])
+		case "Ip6InDiscards":
+			self.IP.InDiscards, _ = strtoull(fields[1])
```

Comment:
nit: InDiscards is sorted above OutRequests in the struct definition, so maybe move this block above the OutRequests case